src/MyEventEmitter.js

```diff
@@ -1,12 +1,57 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(event, listener) {
+    this.events[event] = this.events[event]
+      ? [...this.events[event], listener]
+      : [listener];
+  }
+
+  once(event, listener) {
+    const onceWrapper = (...args) => {
+      listener(...args);
+      this.off(event, onceWrapper);
```

Comment:
The 'once' method currently removes the wrapper function, not the original listener. Consider storing a reference to the wrapper function in the original listener to fix this.